pipeline/oralfacial_analysis.py

```diff
@@ -519,6 +519,230 @@ def make(self, key):
             print(unit_key)
             
         self.insert(units_glm, ignore_extra_fields=True)
+        
+@schema
+class GLMFitNoLick(dj.Computed):
+    definition = """
+    -> ephys.Unit
+    ---
+    r2_nolick: mediumblob
+    weights_nolick: mediumblob
+    y_nolick: longblob
+    predict_y_nolick: longblob
+    x_nolick: longblob
+    """
+    # mtl sessions only
+    key_source = experiment.Session & v_tracking.TongueTracking3DBot & experiment.Breathing & v_oralfacial_analysis.WhiskerSVD & ephys.Unit & 'rig = "RRig-MTL"'
```

Comment:
```suggestion
    key_source = ephys.ProbeInsertion & ephys.Unit & (experiment.Session & v_tracking.TongueTracking3DBot & experiment.Breathing & v_oralfacial_analysis.WhiskerSVD & 'rig = "RRig-MTL"')
```
This table as its primary key referencing the `ephys.Unit`, and I see you have the key_source on the level of `Session`. I'd suggest setting the key_source at the level of ProbeInsertion instead of Session, for 2 reasons:
1. more granularity in parallelization, e.g. 3 workers can work on 3 probe insertions from the same session instead of just one working on that session and the other 2 being idle
2. different probe insertions from the same session may be sufficiently different, hence requiring more specific logic in the `make()`. A prime example is the filtering of units based on some criteria depending on which brain region the probe insertion is targeted. In your case you know that the sessions you're working on are all from one specific brain region, so you can fix the criteria in the `make()`. But to make this table more robust, you can parameterize this (see my example below)